public/app/core/components/NavBar/NavBarItem.tsx

```diff
@@ -73,6 +74,11 @@ const NavBarItem = ({ isActive = false, className, reverseMenuDirection = false,
           disabledKeys={['divider', 'subtitle']}
           aria-label={section.text}
           onNavigate={onNavigate}
+          emptyMessage={
+            link.id === 'starred' ? (
+              <Trans id="nav.empty-starred">Your starred dashboards will appear here</Trans>
+            ) : undefined
+          }
```

Comment:
**Suggestion:** hmm, I wonder if we should extend the [`NavLink`](https://github.com/grafana/grafana/blob/9a85a2e4410266270a1cd59db1937c7ba248966e/pkg/api/dtos/index.go#L59) instead to have this information, so we can keep component more generic?

My reasoning would be that this change would introduce a "specificity" in a generic component. By extending the `NavLink` and putting these information there, we could just do the following on the frontend for example:
```javascript
emptyMessage={ link.emptyMessage ? (
    <Trans id={link.emptyMessageId}>{link.emptyMessage}</Trans>
  ) : undefined
}
```